src/Core/Managed/Shared/Channel/InMemoryTransmitter.cs

```diff
@@ -94,7 +94,7 @@ private void Runner()
                 while (this.enabled)
                 {
                     // Pulling all items from the buffer and sending as one transmissiton.
-                    this.DequeueAndSend();
+                    this.DequeueAndSend(timeout: default(TimeSpan)); // use default timeout
```

Comment:
Won't the default of TimeSpan be zero?
At first I read the comment as meaning that the system has some non-zero timeout value it uses by default which is different than what the code does so it might be clearer to add to the comment so it reads "// Use default timeout of 0."